src/apple.rs

```diff
@@ -203,7 +200,8 @@ pub fn validate_apple_subscription(response: &AppleResponse) -> PurchaseResponse
 /// Validates that a package status is valid
 #[allow(clippy::must_use_candidate)]
 pub fn validate_apple_package(response: &AppleResponse, transaction_id: &str) -> PurchaseResponse {
-    let valid = response.status == 0;
+    let product_id = response.get_product_id(transaction_id);
+    let valid = response.status == 0 && product_id.is_some();
```

Comment:
lets make the 0 a constant that expresses its meaning, maybe with a link to a source where its documented